src/lib/components/filter-select.svelte

```diff
@@ -13,5 +15,22 @@
     on:change
   >
     <slot />
+    <span
```

Comment:
I don't think this does anything.

Reply:
this was most likely left over code i forgot to remove this is like iteration 4 of this menu